.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+  "root": true,
+  "env": {
+    "es6": true
+  },
+  "plugins": [
+    "@typescript-eslint",
+    "import"
+  ],
+  "extends": [
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "rules": {
+    "arrow-body-style": ["error", "as-needed"],
+    "no-debugger": 2,
+    "no-console": 0,
+    "no-empty": "error",
+    "@typescript-eslint/explicit-function-return-type": "error",
+    "@typescript-eslint/no-floating-promises": "error",
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
This rule definitely needs to stay on. If you need help finding the correct types, hit me up.

Reply:
turned it back on

Reply:
Great thanks. Sorry about that, but this rule is very important to keep the maintainability of the code high.